habitat_baselines/common/logging.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+import os
+
+from habitat.core.logging import HabitatLogger
+
+logger = HabitatLogger(
+    name="habitat_baselines",
+    level=logging.INFO
+    if os.environ.get("HABITAT_BASELINES_LOG", 0)
```

Comment:
It hink there should be more levels of logging. Or do we enforce we only ever use info and error everywhere in the code? 
We should use the levels defined here :
https://docs.python.org/3/library/logging.html#logging-levels 



CRITICAL | 50
-- | --
ERROR | 40
WARNING | 30
INFO | 20
DEBUG | 10
NOTSET | 0